cmd/fleet/cron.go

```diff
@@ -225,6 +227,10 @@ func cronVulnerabilities(
 			level.Debug(logger).Log("vulnAutomationEnabled", vulnAutomationEnabled)
 
 			recentVulns := checkVulnerabilities(ctx, ds, logger, vulnPath, config, (vulnAutomationEnabled != ""))
+
+			// TODO: merge results
+			checkOvalVulnerabilities(ctx, ds, logger, vulnPath, config)
```

Comment:
Let's add a TODO comment that this is currently not checking vulnerabilities, it is currently syncing the OVAL definitions (maybe you mean that by "merge results", but I don't follow :).

Reply:
Thanks, I think that TODO is fine we are merging to a feature branch, not main. The final version of `checkOvalVulnerabilities` will also perform the checking.